pola/management/commands/import_brands.py

```diff
@@ -0,0 +1,86 @@
+import csv
+import re
+
+from django.core.management import BaseCommand
+
+from company.models import Brand, Company
+from product.models import Product
+
+
+def update_product(self, brand, ean_code, company, product_name):
+    if Product.objects.filter(code=ean_code).count():
+        product = Product.objects.get(code=ean_code)
+        product.brand = brand
+        product.name = product_name
+        product.save()
+        self.stdout.write(self.style.SUCCESS(f"Successfully updated product {product_name}"))
+    else:
+        Product(company=company, brand=brand, code=ean_code, name=product_name).save()
+        self.stdout.write(self.style.SUCCESS(f"Successfully created product {product_name}"))
+
+
+class Command(BaseCommand):
+    help = 'Import lidl companies from .tsv file'
+
+    def add_arguments(self, parser):
+        parser.add_argument('tsv_filepath')
+
+    def handle(self, *args, **options):
+        if options['tsv_filepath']:
+            with open(options['tsv_filepath']) as csv_file:
+                csv_reader = csv.reader(csv_file, delimiter='\t')
+                line_count = 0
+                index_successful = 0
+                for row in csv_reader:
```

Comment:
```suggestion
                for line_no, row in enumerate(csv_reader):
```
Takie podejście jest zalecane w Python, gdy nie ma masz potrzeby manipulowaania dodatkowo tym indexem.